### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by Kilvoctu** [Kilvoctu/aiyabot#229] [Announcements]
Edit Aug 2026:
----------------

As fond as I am for this project, it was a lot of fun and I learned much, I don't think there's any reason for me to return. I've moved on to other ML adjacent projects since. Additionally, as we've been in the "vibecoding" era for a while now, anyone can easily fork the source and do whatever they want with it with no difficulty. So I'm gonna go ahead and archive the repo.

Original below
----------------

Heya,
Sorry I'm not following and keeping up to date on this project much. I do genuinely feel bad about it.

I love my aiyabot greatly and work on this project for fun. There was a lot of initial excitement in the first several months, but over time I log into Github and see many complaints and problems. The negative feelings I got just vastly overshadowed any positive reinforcement.
In addition, I'm not currently in a life position to spend much time on this, or even SD stuff in general.. If/when my life stabilize, I would like to continue development, but I don't know when that'll be as it's been several months already.

Thank you for your understanding.

**Comment by solareon** [comment]
If you would like me to help out with maintaining I am more than happy to help. I understand that life gets in the way sometimes. This is a really cool piece of software and would love to be a part of keeping it going.

**Comment by wizz13150** [comment]
Hey Sir !

Your bot is just awesome ! I'm still actively using [it](https://discord.gg/dmcSRAjqmt).
My [version](https://github.com/wizz13150/aiyabot/tree/DiscordFullBot) is kind of messy, lazy too. Sad i didn't do things right to post a PR for each part separately...

![image](https://github.com/Kilvoctu/aiyabot/assets/22177081/12dd012d-8e3e-4b79-97b9-978144030209)

Just take care and take your time. It's the important part to focus on, for sure.
Hakuna Matata, Much love on you ! 💖

And thanks to Solareon too, active as well to help people here. I would trust him to maintain this repo.

Cheers 🥂

**Comment by wizz13150** [comment]
Hey Sir ! (:

Happy to see some commits here.

I would be happy to discuss with you further and show you my current state on discord, regarding the bot, as i basically stealed every good commits in the forks.

Solareon is also welcome, or any peeps with ideas.

Current state of /Generate : 
![image](https://github.com/Kilvoctu/aiyabot/assets/22177081/90f9ed4d-7ce9-43d1-83b2-5d7688eb1823)

Progression message with Live Preview: 
![image](https://github.com/Kilvoctu/aiyabot/assets/22177081/e6f4df1f-90a7-4398-a5c1-6f15f6ba1cc6)

ADetailer, Deforum, CN [...]
No fundamental changes, only new features.
My code is messy, but some very cool features and pretty amazing pics.

Cheers ! 🥂
